examples/docs/en-US/table.md

```diff
@@ -1183,7 +1183,8 @@ Customize table column so it can be integrated with other components.
 | fit | whether width of column automatically fits its container | boolean | — | true |
 | show-header | whether table header is visible | boolean | - | true |
 | highlight-current-row | whether current row is highlighted | boolean | — | false |
-| row-class-name | function that returns custom class names for a row | Function(row, index) | — | — |
+| row-class-name | function that returns custom class names for a row. If its value is a string, the value will be every row's class name | Function(row, index)/String | — | — |
+| row-style | function that returns custom style for a row. If its value is a object, the value will be every row's style | Function(row, index)/Object | — | — |
```

Comment:
`If its value is a object, the value will be every row's style`
=>
`If the returned value is an object, it will apply to every row`